resources/warnings.json

```diff
@@ -4151,7 +4151,8 @@
     "url": "https://jenkins.io/security/advisory/2019-04-03/#SECURITY-835",
     "versions": [
       {
-        "pattern": ".*"
+        "lastVersion": "1.25",
+        "pattern": "(1[.][0-2][0-5])"
```

Comment:
This pattern would not consider 1.3 through 1.9, or 1.16 through 1.19 to be vulnerable.
```suggestion
        "pattern": "(1[.]1[0-9]|1[.]2[0-5]|1[.][3-9])(|[.-].+)"
```

Reply:
Thank you very much @daniel-beck for the suggestion and apologies as this slipped through my notifications.
This looks good to me. Accepting the change and merging.